src/mount/mastercomm.cc

```diff
@@ -1326,6 +1341,10 @@ void fs_term(void) {
 	threchead = nullptr;
 	rec_lock.unlock();
 	std::unique_lock<std::mutex> af_lock(acquiredFileMutex);
+#ifdef _WIN32
+	acquiredFilesLastTimeUsed.clear();
```

Comment:
```suggestion
	acquiredFiles.clear();
```

Reply:
Considered and commented [here ](https://github.com/leil-io/saunafs/pull/230#discussion_r1872378478)